### DISCUSSION TITLE
South Pole

### DISCUSSION THREAD

**Discussion by zeze1959** [Stellarium/stellarium#2397] [Q&A] [answered]
I have a question to ask because it confuses me. I have a location created on the South Pole and when looking at the sky the stars appear in a different position than I see on other maps of the sky for the same area.
For example in the Stellarium when I´m facing South the star Sirius appears to the east but on other maps it is always to the West.
Did anyone knows why?

**Comment by alex-w** [comment, marked as answer]
You cannot facing South on the South Pole
